src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -103,6 +103,12 @@ IPropertySymbol property
 			return InspectConcreteType( type, typesInCurrentCycle, flags );
 		}
 
+		public MutabilityInspectionResult InspectMember( ISymbol symbol ) {

```

Comment:
Common pattern for `public` methods of this class is to hide the infinite-loop detector.